### PR TITLE
License

### PR DESCRIPTION
**Pull request by bastien-roucaries**
Here some work.

I put your contribution under zlib license for zlib code. zlib license is compatible with expat you use but more restrictive... So putting under zlib the combined work is safe.

You contribution are still under expat but not the combined work.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,14 +11,16 @@
   "homepage": "https://github.com/nodeca/pako",
   "contributors": [
     "Andrei Tuputcyn (https://github.com/andr83)",
-    "Vitaly Puzrin (https://github.com/puzrin)"
+    "Vitaly Puzrin (https://github.com/puzrin)",
```
Comment (puzrin) [inline package.json]: Will it be ok, if i drop this field entirely? I suspect, it was done for another things, and now it's just a one more point of maintenance pain. If we start add all participators here, we need to add everyone who did PR into paco. But this page is much more informative and useful https://github.com/nodeca/pako/graphs/contributors

File: lib/zlib/adler32.js [hunk 2]
```diff
@@ -3,6 +3,25 @@
 // Note: adler32 takes 12% for level 0 and 2% for level 6.
 // It doesn't worth to make additional optimizationa as in original.
 // Small size is preferable.
+//
+// (C) 1995-2013 Jean-loup Gailly and Mark Adler
+// (C) 2014-2017 Vitaly Puzrin and Andrey Tupitsin
+//
+// This software is provided 'as-is', without any express or implied
```
Comment (puzrin) [inline lib/zlib/adler32.js]: I have no principal objections, but is it necessary to add such big text to every header (from this line and below)? Looks boring. Even original zlib sources don't have such thing.

File: package.json (hunk 1 above)
Comment (bastien-roucaries) [inline package.json]: Depends. In europe particularly in civil law system you should add author except for minor contrbutions... Safer to do before distribution

File: lib/zlib/adler32.js (hunk 2 above)
Comment (bastien-roucaries) [inline lib/zlib/adler32.js]: Original source are made by original author that are not binded by license.... 

You could strip at build time if needed

**Comment by puzrin** [comment]
And the last question. Will you allow me to do final cummulative commit from my name instead of accepting this PR? I'd like license-related updates to be done directly by me & in single commit.

**Comment by bastien-roucaries** [comment]
Do as you like. BTW what I have done is not copyrightable. So feel free to do like you want

**Comment by puzrin** [comment]
Thanks! I was once in unpleasant situation after incorporated PR with minor (from my point of view) readme changes to another more significant commit. Now prefer to ask permission for sure :)

**Comment by puzrin** [comment]
Close in favour of https://github.com/nodeca/pako/commit/eaead04de324863b753293589e5fb78fc9701769

**Pull request closed without merging**